lib/tasks/osc.rake

```diff
@@ -74,6 +74,21 @@ namespace :osc do
     end
   end
 
+  def check_IDs_in_changes_file!
+    # Checking if any new bugzilla,fate,... IDs are defined
+    # in the *.changes file
+    puts "Checking IDs in *.changes file" if verbose
+    Dir.chdir(osc_checkout_dir) do
+      cmd = "osc -A '#{obs_api}' diff *.changes | grep \"bnc#[0-9]\\|fate#[0-9]\\|boo#[0-9]\\|bsc#[0-9]\\|bgo#[0-9]\""
+      puts cmd if verbose
+      `bash -c '#{cmd}'`
+      return if $?.success?
+    end
+    puts "Stopping, missing new bugzilla or fate entry in the *.changes file"
+    puts "Valid entries are bnc#<number>, fate#<number>, boo#<number>, bsc#<number>, bgo#<number>"
+    exit 0
```

Comment:
no, `Stop commiting, no difference from devel project` is success of script. Not a error. It just say that there is no change from devel project, so often rerun of jenkins due to some worker restart or so on. So result is correct as devel project contain actual sources.
This is different case as it report that something is wrong with sources and jenkins/travis should report error as something is wrong. That is also reason why I suggest to put it into check tasks